woocommerce/woocommerce-android
# Screen for a merchant to choose connection to bluetooth or built in reader

Closes: #8134


### Description

If the feature flag is ON, then we show the dialog where a merchant can choose either BT reader or built-in reader

### Testing instructions

With the Feature flag ON
* Start payment flow
* Click CARD
* Notice the new dialog
* Click on "tap to pay". Notice that nothing happens (not implemented yet)
* Click on "BT reader". Notice that the usual IPP flow goes on

With the Feature flag OFF
The current flow should be available only (not reader type picker should be accessible)

### Images/gif



https://user-images.githubusercontent.com/4923871/212274802-71a36ce9-35c0-40a2-b091-80c8ded3dcc2.mp4
